source/jormungandr/jormungandr/parking_space_availability/car/common_car_park_provider.py

```diff
@@ -84,21 +84,14 @@ def _call_webservice(self, request_url):
             self.record_call("OK")
             return data.json()
```

Comment:
You should call `data.json()` before doing the `record_call` as this method can raise an exception: http://docs.python-requests.org/en/master/user/quickstart/#json-response-content